docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -52,6 +50,9 @@ Follow these steps for a new or existing project to configure it to use a shared
     <DefaultSettings>
     ```
 
+> [!NOTE]
+> If your add-in is an Excel add-in with custom functions, be sure to remove the `CustomFunctionsRuntime` requirement.
+
 1. Find the `<VersionOverrides>` section and add the following `<Runtimes>` section. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the ` <bt:Urls>` section near the bottom of the **manifest.xml** file.
 
     > [!IMPORTANT]
```

Comment:
The first step in the "Configure the webpack.config.js file" section should say something like "...and open the add-in project you generated."

Reply:
Fixed.